build.rs

```diff
@@ -35,27 +36,34 @@ fn clang_args() -> Vec<String> {
     flags
 }
 
-#[cfg(not(any(feature = "gecko", target_os = "windows")))]
+#[cfg(not(feature = "gecko"))]
 const fn clang_args() -> Vec<String> {
     Vec::new()
 }
 
-#[cfg(not(windows))]
 fn bindgen() {
-    #[cfg(target_os = "linux")]
-    let bindings = bindgen::Builder::default()
-        .header_contents("rtnetlink.h", "#include <linux/rtnetlink.h>")
-        // Only generate bindings for the following types
-        .allowlist_type("rtattr|rtmsg|ifinfomsg|nlmsghdr");
-    #[cfg(not(target_os = "linux"))]
-    let bindings = bindgen::Builder::default()
+    let target_os = env::var("CARGO_CFG_TARGET_OS").expect("CARGO_CFG_TARGET_OS was not set");
```

Comment:
Note `target_os` here is the target OS of the final binary, not the target os of the `build.rs` binary.